themes/edx.org/lms/templates/header.html

```diff
@@ -104,6 +104,10 @@ <h2 class="course-header">
 
       <%include file="user_dropdown.html"/>
 
+      <a href="${get_online_help_info(online_help_token)['doc_url']}" 
```

Comment:
For a11y there really shouldn't be lists that only have 1 item because the screenreaders are so verbose around describing lists.